src/odemis/driver/hamamatsurx.py

```diff
@@ -1000,6 +1007,11 @@ def terminate(self):
         for child in self.children.value:
             child.terminate()
 
+        try:
+            self.AppEnd()
+        except:
+            logging.info("Failed to stop the App", exp_info=True)
```

Comment:
Sorry, turns out that the right spelling is `exc_info`.

Reply:
ah ok, it is (force) updated now